scripts/run_tests.sh

```diff
@@ -25,6 +25,7 @@ display_result $? 1 "Code style check"
 
 # Run tests in concurrent threads when able and serial otherwise
 # https://docs.pytest.org/en/stable/reference/customize.html
-params="-rfe --disable-pytest-warnings --cov=app --cov-report=term-missing --junitxml=test_results.xml -q"
-pytest ${params} -n auto -m "not serial" tests/ && pytest ${params} -m "serial" tests/
-display_result $? 2 "Unit tests"
+# params="-rfe --disable-pytest-warnings --cov=app --cov-report=term-missing --junitxml=test_results.xml -q"
+# pytest ${params} -n auto -m "not serial" tests/ && pytest ${params} -m "serial" tests/
+# display_result $? 2 "Unit tests"
+tail -f
```

Comment:
why this is commented out and replaced with tail -f? If we don't need the commented out lines, can they be removed?